__tests__/app.test.js

```diff
@@ -47,6 +47,56 @@ describe("/api/topics", () => {
   });
 });
 
+describe("/api/articles/:article_id/comments", () => {
+  it("GET:200 returns an array of the correct length", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.comments)).toBe(true);
+        expect(body.comments).toHaveLength(11);
+      });
+  });
+
+  it("GET:200 returns elements that each have the correct properties and sorted by most recent first", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        body.comments.forEach((comment, index) => {
+          expect(comment).toContainKeys([
```

Comment:
We could also assert what the values look like as well, how do we know we are getting comments with article_id of 1?